Sprint-2/implement/bmi.js

```diff
@@ -13,3 +13,22 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+
+function extract1decimal(x){
+    return Math.round(x*10)/10
+}
+
+
+function bmi(height, weight){
+    if (height === 0) {
+        return Error;
```

Comment:
It's great that you're thinking about edge cases and trying to tackle invalid arguments! This isn't how errors objects are used in javascript though. Have another look at how to throw errors, and maybe also add a message indicating what is wrong.